### DISCUSSION TITLE
[Q&A] Power Platform Developer Office Hours Recording

### DISCUSSION THREAD

**Discussion by sanjeev40084** [microsoft/powerplatform-build-tools#633] [Q&A]
Does anyone know where Power Platform Developer Office Hours get posted?

**Comment by snizar007** [comment]
Hello @sanjeev40084 - I will be posting on YouTube (same location as last month,  https://youtu.be/5H0qMU73V2A?si=yM9M5rsF2fbS4Sjx).
